backend/app/repositories/base.py

```
@@ -0,0 +1,290 @@
+
+            # Update timestamp
+            kwargs['updated_at'] = datetime.utcnow()
+
+            for field, value in kwargs.items():
+                if hasattr(instance, field):
+                    setattr(instance, field, value)
+
+            await self.session.commit()
+            await self.session.refresh(instance)
+            logger.info(f"Updated {self.model.__name__} with id {entity_id}")
+            return instance
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)